backend/fms_core/template_importer/row_handlers/library_qc/library_qc.py

```diff
@@ -0,0 +1,116 @@
+
+
+
+from fms_core.models.instrument_type import InstrumentType
+from fms_core.utils import convert_concentration_from_nm_to_ngbyul
+from fms_core.services.sample import update_sample, get_sample_from_container
+from fms_core.services.process_measurement import create_process_measurement
+from fms_core.services.property_value import create_process_measurement_properties
+from fms_core.services.library import set_library_size
+from fms_core.template_importer.row_handlers._generic import GenericRowHandler
+
+INSTRUMENT_PROPERTIES = ['Library Quality Instrument', 'Library Quantity Instrument']
+QC_PLATFORM = "Quality Control"
+
+class LibraryQCRowHandler(GenericRowHandler):
+    def __init__(self):
+        super().__init__()
+
+    def process_row_inner(self, sample_container, measures, process, process_measurement, process_measurement_properties):
+         # Get the library sample that was checked
+        source_sample_obj, self.errors['container'], self.warnings['container'] = \
+            get_sample_from_container(barcode=sample_container['container_barcode'], coordinates=sample_container['container_coord'])
+
+        if source_sample_obj is None:
+            self.errors['sample_source'] = 'Library sample for QC was not found at the specified container.'
```

Comment:
You might wanna include the container name or barcode to provide more information to the user. i.e. `at the specified container {container.barcode}` or something like that